packages/api/tests/controllers/nomination.test.js

```diff
@@ -15,20 +17,37 @@ describe('GET Nomination Endpoint', () => {
   });
 
   it('returns a 200 when nomination exist', async () => {
-    const res = await request(app)
-      .get(`/nomination/${nomination.id}`);
+    const res = await request(app).get(`/nomination/${nomination.id}`);
     expect(res.statusCode).toBe(200);
   });
 
   it('return a 404 when nomination does not exist', async () => {
-    const res = await request(app)
-      .get('/nomination/00000000-0000-0000-0000-000000000000');
+    const res = await request(app).get(
+      '/nomination/00000000-0000-0000-0000-000000000000'
+    );
     expect(res.statusCode).toBe(404);
   });
 
   it('return a 400 when uuid is not a valid uuid', async () => {
-    const res = await request(app)
-      .get('/nomination/326');
+    const res = await request(app).get('/nomination/326');
     expect(res.statusCode).toBe(400);
   });
 });
+
+describe('Create Nomination Endpoint', () => {
+  it('returns 400 when field is not valid', async () => {
+    const data = {
+      id: null,
+    };
+    const res = await request(app).post('/nomination').send(data);
+    expect(res.statusCode).toBe(400);
+  });
+
+  it('return 201 when nomination is created', async () => {
+    const data = {
+      providerFirstName: 'test provider',
```

Comment:
Didn't we change this to `providerName` in the last PR?

Reply:
yes! Looking through, should I go through nomination model and change most fields to  !allowNull?

Reply:
@hchamorro I think it probably makes sense to set `allowNull: false` on more fields, but I'm not sure off-hand which fields we want to require, and which ones are fine to `allowNull` on.

@JigSays Do you know which Nomination fields we want to require and which are optional? If not, we may be able to clean this up in a follow-up.

Reply:
@H12, right now KSF has ALL fields marked as required.

Reply:
@JigSays Thanks!

There's your answer, @hchamorro: `allowNull: false` _all_ the things.

Reply:
sweeeeeeeeet!........................................ ;'(